src/util/records.py

```diff
@@ -1047,13 +1047,27 @@ def replace_record_references_batch(cr, id_mapping, model_src, model_dst=None, r
                 cols = get_columns(cr, table, ignore=(fk,))
```

Comment:
Use the new toys.
```suggestion
                cols = get_columns(cr, table, ignore=(fk,)).iter_unquoted()
```

Then, the `get_fk()` call above should pass the `quote_ident=False` argument, and we should use `format_query()` instead of `query.format()`